client/Components/Hello.ts

```diff
@@ -2,7 +2,15 @@
 
 import * as React from 'react';
 
-export default class HelloMessage extends React.Component {
+interface HelloProps{
+    name: string;
+}
+
+interface HelloState{
+    
+}
+
+export default class HelloMessage extends React.Component<HelloProps, HelloState> {
```

Comment:
This wasn't working because of the missing `<HelloProps, HelloState>` and above interface definitions.